netdissect.h

```diff
@@ -700,6 +700,7 @@ extern void rpki_rtr_print(netdissect_options *, const u_char *, u_int);
 extern void rrcp_print(netdissect_options *, const u_char *, u_int, const struct lladdr_info *, const struct lladdr_info *);
 extern void rsvp_print(netdissect_options *, const u_char *, u_int);
 extern int rt6_print(netdissect_options *, const u_char *, const u_char *);
+extern int rt6_tlv_print(netdissect_options *, const u_char *, u_int);
```

Comment:
No need to declare `rt6_tlv_print` in  netdissect.h. It is only used in `print-rt6.c` where it should be `static`.